packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -1,49 +1,175 @@
+
+            <f-link
+                is-distinct
+                :class="$style['c-accountInfo-customerCareLink']"
+                href="/help"
+                target="_blank">
+                {{ $t('contactCustomerCareTeam') }}
+            </f-link>
+
+            <form-field
+                :label-text="$t('fields.firstNameLabel')"
+                :placeholder="$t('fields.firstNamePlaceholder')"
+                :value="fields.firstName" />
+
+            <form-field
+                :label-text="$t('fields.lastNameLabel')"
+                :placeholder="$t('fields.lastNamePlaceholder')"
+                :value="fields.lastName" />
+
+            <h2
+                class="u-spacingBottom--large">
+                {{ $t('deliveryAddress') }}
+            </h2>
+
+            <form-field
+                :label-text="$t('fields.addressLabel')"
+                :placeholder="$t('fields.line1Placeholder')"
+                :value="fields.line1" />
+
+            <form-field
+                :placeholder="$t('fields.line2Placeholder')"
+                :value="fields.line2" />
+
+            <form-field
+                :placeholder="$t('fields.line3Placeholder')"
+                :value="fields.line3" />
+
+            <form-field
+                :label-text="$t('fields.cityLabel')"
+                :placeholder="$t('fields.cityPlaceholder')"
+                :value="fields.city" />
+
+            <form-field
+                :label-text="$t('fields.postcodeLabel')"
+                :placeholder="$t('fields.postcodePlaceholder')"
+                :value="fields.postcode" />
+
+            <f-button
+                :class="[$style['c-accountInfo-submitButton']]"
+                data-test-id="account-info-submit-button"
+                button-type="primary"
+                button-size="large"
+                is-full-width
+                action-type="submit">
+                {{ $t('buttons.saveChanges') }}
+            </f-button>
+        </form>
+
+        <f-button
+            :class="[$style['c-accountInfo-changePasswordButton']]"
+            data-test-id="account-info-submit-button"
+            button-type="secondary"
+            button-size="large"
+            is-full-width
+            action-type="submit">
+            {{ $t('buttons.changePassword') }}
+        </f-button>
+
+        <hr>
+
+        <p>{{ $t('deleteAccountMessage') }}</p>
+
+        <f-link
+            is-distinct
+            href="/help"
+            target="_blank">
+            {{ $t('deleteAccountLink') }}
+        </f-link>
+    </card>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
-import tenantConfigs from '../tenants';
+import { VueGlobalisationMixin } from '@justeat/f-globalisation';
 
-// eslint-disable-next-line no-unused-vars
-import AccountInfoServiceApi from '../services/AccountInfoServiceApi';
+import Card from '@justeat/f-card';
+import '@justeat/f-card/dist/f-card.css';
```

Comment:
Better to load in the CSS as part of the style block

Reply:
Worth updating the readme's for the atoms - I followed these

I would put in the style block in coreweb, but didn't in fozzie as I didn't expect it would handle it

Reply:
Actually, I think you're right. It doesn't actually matter in the component, it's in CoreWeb it matters as that's where it gets extracted. So you're fine to leave it here in the component itself

Reply:
I have moved it to styles and it still works; so probably worth us updating the readme for the atoms to get consistency as we can

Reply:
FYI: works locally when in styles; but not circleCI

![image](https://user-images.githubusercontent.com/10741583/142633026-240b5984-89fb-429d-b863-2b4707cb3df0.png)